recipes/trust4/meta.yaml

```diff
@@ -0,0 +1,39 @@
+package:
+  name: trust4
+  version: 1.0.2.beta
+
+build:
+  number: 0
+
+source:
+  url: https://github.com/liulab-dfci/TRUST4/archive/v1.0.2-beta.tar.gz
+  sha256: 81c40911249d541aa20d68d114cb2246fd7e0c3351bfafd74c26597aee170201
+
+requirements:
+  build:
+    - {{ compiler('cxx') }}
+    - {{ compiler('c') }}
+  host:
+    - samtools 0.1.19.*
+    - zlib
+  run:
+    - zlib
+    - perl
+
+test:
+  commands:
+    - trust4 2>&1 | cat > /dev/null
+    - fastq-extractor 2>&1 | cat > /dev/null
+    - bam-extractor 2>&1 | cat > /dev/null
+    - annotator 2>&1 | cat > /dev/null
+    - which trust-simplerep.pl
+    - which trust-barcoderep.pl
+    - which trust-smartseq.pl
+    - which BuildDatabaseFa.pl
+    - which BuildImgtAnnot.pl
+    - which run-trust4
+
+about:
+  home: https://github.com/liulab-dfci/TRUST4 
+  summary: TCR and BCR assembly from bulk or single-cell RNA-seq data
+  license: GPLv3
```

Comment:
please add a `license_file` and use ` GPL-3.0-only` if its it GPLv3 only.

Reply:
It is updated. Thank you!